psl/psl/tests/config/sources.rs

```diff
@@ -730,26 +730,7 @@ fn relation_mode_default() {
 
         generator client {
           provider = "prisma-client-js"
-          previewFeatures = ["relationMode"]
-        }
-    "#};
-
-    let config = parse_configuration(schema);
-
-    assert_eq!(config.relation_mode(), Some(RelationMode::ForeignKeys));
-}
-
-#[test]
-fn relation_mode_and_referential_integrity_can_coexist() {
-    let schema = indoc! {r#"
-        datasource ps {
-          provider = "sqlserver"
-          url = "mysql://root:prisma@localhost:3306/mydb"
-        }
-
-        generator client {
-          provider = "prisma-client-js"
-          previewFeatures = ["referentialIntegrity", "relationMode"]
+          previewFeatures = ["referentialIntegrity"]
         }
     "#};
```

Comment:
If I'm reading the diff correctly, it looks like the contents of this test are exactly the same as that of the `referential_integrity_default` test above.

Reply:
I'll remove `referential_integrity_default`